### DISCUSSION TITLE
netCDF4

### DISCUSSION THREAD

**Discussion by jarrodcolburn** [python/typing#1335] [General]
reference netCDF4 package....

This is less of a discussion, and more a post for posterity for people who may wish to generate netCDF4 types, and want to carry on where I'm leaving off...
https://github.com/jarrodcolburn/typeshed/tree/netcdf4/stubs/netCDF4
my plan was to create a PR,  but unfortunately, I had to move on to other projects, but maybe someone else can use this.

**Comment by srittau** [comment]
Thank you for your work so far! I have opened python/typeshed#9545 so that this does not get lost.
